src/solver/multiphysics_integrator.h

```diff
@@ -336,7 +336,7 @@ namespace solver
         virtual double getMaxFinerLevelDt(const int finerLevelNumber, const double coarseDt,
                                           const SAMRAI::hier::IntVector& ratio) override
         {
-            return std::pow(coarseDt, ratio.max());
+            return coarseDt / (ratio.max() * ratio.max()) / 2.;
```

Comment:
no changes requested, just adding for posterity:
https://stackoverflow.com/questions/18165047/multiply-by-0-5-rather-than-divide-by-2